include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.cpp

```diff
@@ -336,14 +336,18 @@ create_geometric_coarsening_sequence(Triangulation<dim, spacedim> const & fine_t
     parallel::distributed::Triangulation<dim, spacedim> temp_tria(
       fine_triangulation->get_communicator(), fine_triangulation->get_mesh_smoothing());
 
+    MPI_Comm mpi_comm = fine_triangulation_in.get_communicator();
     temp_tria.copy_triangulation(*fine_triangulation);
+    temp_tria.coarsen_global();
 
-    // create coarse meshes
-    for(unsigned int l = (fine_triangulation->n_global_levels() - 1); l > 0; --l)
-    {
-      // coarsen mesh
-      temp_tria.coarsen_global();
+    unsigned int min_level = fine_triangulation->n_global_levels() - 1;
```

Comment:
why is this variable called `min_level` and not just `level`, i.e., it is of type `unsigned int` (no `const`)?